src/main/kotlin/no/digdir/fdk/searchservice/model/SearchOperation.kt

```diff
@@ -10,13 +10,13 @@ data class SearchOperation(
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 data class SearchFilters(
-  val opendata: Boolean?,
-  val accessRights: String?,
-  val theme: String?,
-  val spatial: String?,
-  val provenance: String?,
-  val los: String?,
-  val orgPath: String?,
+  val opendata: SearchFilter<Boolean>?,
+  val accessRights: SearchFilter<String>?,
+  val theme: SearchFilter<String>?,
```

Comment:
Når du først er i gang, endre denne og fjern støtten av komma-separert verdi
```suggestion
  val theme: SearchFilter<List<String>>?,
```